### ISSUE TITLE
Ctrl+TAB for switching between files opened in two tabs freezes both tabs (requires manual kill of nqq)

### ISSUE THREAD

**Issue by tudorp** [notepadqq/notepadqq#992]
**Description**
Ctrl+TAB for switching between files opened in two different tabs freezes both tabs (can't scroll/edit any of them), although the rest of the UI seems unaffected.

Attempting to close the tabs does not produce any results.

Attempting to close nqq seems to work, but in fact the process remains active, just the UI is hidden and on restart it resumes in exactly same state. Restoring functionality requires to manually terminate the nqq parent process.
#
**To Reproduce**
1. Start nqq (e.g. using system search)
2. Open an existing file (e.g. python script)
3. Open another existing file (e.g. another python script)
4. Press Ctrl+TAB to switch between the tabs
5. Observe that the tab switched can't be scrolled/edited (switching back to previous tab by shorcut or UI finds it also frozen)
6. Rest of UI appears working, but the tabs can't be closed
7. Closing nqq seems successful, but on restart same frozen tabs are observed (actually nqq process does not stop on close...)
8. Stopping nqq by kill/killall/htop apps makes it possible afterwards to restart successfully nqq

**Expected behavior**
At step 5 it is expected that editing/scroll is possible
At step 6 it is expected for the tabs to be closed
At step 7 it is expected that all nqq processes are stopped on exit and functionality restored on fresh start
 
**Environment**
 - OS: Linux Mint 20.1 Cinemon 4.8.6 kernel 5.8.0-45-generic
 -  ##installed notepadqq 2.0.0~beta1-1build1 from APT-Sources: http://archive.ubuntu.com/ubuntu focal/universe amd64 Packages 
 - from distro packages
 - Qt version 5.12.8.
 - Notepadqq version v2.0.0-beta+git (as displayed in menu)

**Additional context**
The issue was discovering while editing two exiting python source files and _did not reproduce on a simple test case (start nqq with new emty tab, add new tab, switch between tabs with Ctrl+TAB seems to work OK)_

Discovered on freshly installed (same day) of nqq from the APT-Sources: http://archive.ubuntu.com/ubuntu focal/universe amd64 Packages on Linux Mint 20.1 Cinemon 4.8.6 kernel 5.8.0-45-generic.

I find it interesting that opening new tabs (non existing files) does not create the same issue...

**Comment by ricir** [comment]
Same problem.
Also when using Ctrl+PgUp or Ctrl+PgDwn
I found also that clicking on different tabs ends in all tabs disappearing but for one.
Solved only by killing notepadqq.
Very annoying because I always forget the issue and fall in the problem again.

**Comment by OutstandingBillNeal** [comment]
TLDR: Open the Settings | Preferences dialog, select Shortcuts, View, Toggle to Former Tab, then assign the shortcut you want (e.g. Ctrl+Tab).

It seems Ctrl+Tab isn't an assigned keyboard shortcut by default.  Once the shortcut has been assigned, it stops messing things up.

**Comment by tsjnachos117** [comment]
I can confirm this also affects the <kbd>CTRL</kbd>+<kbd>PGUP</kbd> and <kbd>CTRL</kbd>+<kbd>PGDN</kbd> keys, which I find myself habitually using all the time. Strangely, it seems new/empty documents aren't affected. I guess this only happens when you open existing files?

Unfortunately, @OutstandingBillNeal's workaround won't work with these keys, as I can't seem to find a way to forcibly program them.

On a plus note, you don't have to kill NQQ by process id: xkill does the job just fine, which is more convenient for me.

**Comment by Collud2** [comment]
I also experience this crash, with Ubuntu 22.04 Cinnamon 5.2.7 (kernel 6.2.0-39), and also Ubuntu 20 + 22 Mate.
As @tsjnachos117 mentions, there isn't an applicable action for "view to next/previous tab" though this feels like it is missing conspicuously. 
Binding Ctrl+PgUp/Dn to other actions, like "about Qt" eliminates the crash, but is still useless for our purposes.
